#!/usr/bin/env python
PACKAGE = "quanergy_client_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("int_param",    int_t,    0, "Parametro de resolucion", 100,  50, 200)


exit(gen.generate(PACKAGE, "quanergy_client_ros", "resolution"))
